mungegithub/mungers/bulk-lgtm.go

```diff
@@ -17,12 +17,19 @@ limitations under the License.
 package mungers
 
 import (
+	"bytes"
+	"crypto/rand"
+	"encoding/binary"
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"strconv"
+	"strings"
 	"sync"
 
+	"golang.org/x/oauth2"
```

Comment:
oh, yup... you need to add oauth2 to the [BUILD file](https://github.com/kubernetes/test-infra/blob/master/mungegithub/mungers/BUILD#L90)